packages/react-dom/src/__tests__/ReactCompositeComponent-test.js

```diff
@@ -365,22 +357,10 @@ describe('ReactCompositeComponent', () => {
     expect(renders).toBe(1);
 
     instance.setState({value: 1});
-
     expect(renders).toBe(2);
 
     ReactDOM.render(<div />, container);
-
-    expect(() => {
```

Comment:
since we are not asserting anything here now, wouldn't this test pass even if the actual implementation DID throw a warning ?

Reply:
No, our infra catches unexpected warnings.